config.json

```
@@ -727,8 +759,15 @@
         "slug": "nucleotide-count",
         "name": "Nucleotide Count",
         "uuid": "caca1c6a-b998-431e-b6af-ca2d47b7708f",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "dictionaries"
```

Maybe also add `linq`?

```suggestion
          "dictionaries",
          "linq"
```